mParticle-Apple-SDK/Network/MPURLRequestBuilder.m

```diff
@@ -170,16 +170,9 @@ - (NSMutableURLRequest *)build {
                 kits = nil;
             }
             
-            NSArray<id<MPExtensionKitProtocol>> *activeKitsRegistry = [[MParticle sharedInstance].kitContainer activeKitsRegistry];
-            if (activeKitsRegistry.count > 0) {
-                NSMutableArray<NSNumber *> *activeKitIds = [[NSMutableArray alloc] initWithCapacity:activeKitsRegistry.count];
-                
-                for (id<MPExtensionKitProtocol> kitRegister in activeKitsRegistry) {
-                    [activeKitIds addObject:kitRegister.code];
-                }
-                
-                kits = [activeKitIds componentsJoinedByString:@","];
-            }
+            NSMutableArray<NSNumber *> *configuredKitIds = [[NSMutableArray alloc] initWithArray:[[MParticle sharedInstance].kitContainer configuredKitsRegistry]];
```

Comment:
Nitpick, but for consistency with the rest of this file and general readability, I'd prefer to see dot syntax here. 

So `[MParticle sharedInstance].kitContainer` would be `MParticle.sharedInstance.kitContainer`

Reply:
Also this code can be slightly simplified to use `mutableCopy` instead of the whole alloc/init.

So like this: `[MParticle.sharedInstance.kitContainer mutableCopy];`

Reply:
Though be aware that would mean if `kitContainer` or `configuredKitsRegistry` is nil, `configuredKitIds` would be nil, and so `kits` would be nil, whereas this way, `configuredKitIds` will always be an object, so you may not want to use `mutableCopy` if that's required.

Reply:
🙌 will do

Reply:
for my own knowledge, when can you use dot syntax instead of `[]`? I feel like sometimes I try to use the dot syntax and it's fine, but sometimes the XCode linter throws a fit..

Reply:
In general, it should also be possible to use dot syntax anywhere you use bracket syntax, except for when the method takes parameters. I'd have to see a concrete example of where the linter is giving problems.